examples/long-short.py

```diff
@@ -11,328 +12,275 @@
-      if(i < longShortAmount):
-        self.short.append(stockField[0])
-      elif(i > (len(self.allStocks) - 1 - longShortAmount)):
-        self.long.append(stockField[0])
-      else:
-        continue
-
-    # Determine amount to long/short based on total stock price of each bucket.
-    equity = int(float(self.alpaca.get_account().equity))
-
-    self.shortAmount = equity * 0.30
-    self.longAmount = equity - self.shortAmount
-
-    respGetTPLong = []
-    tGetTPLong = threading.Thread(target=self.getTotalPrice, args=[self.long, respGetTPLong])
-    tGetTPLong.start()
-    tGetTPLong.join()
-
-    respGetTPShort = []
-    tGetTPShort = threading.Thread(target=self.getTotalPrice, args=[self.short, respGetTPShort])
-    tGetTPShort.start()
-    tGetTPShort.join()
-
-    self.qLong = int(self.longAmount // respGetTPLong[0])
-    self.qShort = int(self.shortAmount // respGetTPShort[0])
-
-  # Get the total price of the array of input stocks.
-  def getTotalPrice(self, stocks, resp):
-    totalPrice = 0
-    for stock in stocks:
-      bars = self.alpaca.get_bars(stock, TimeFrame.Minute,
-                                  pd.Timestamp('now').date(),
-                                  pd.Timestamp('now').date(), limit=1,
-                                  adjustment='raw')
-
-      totalPrice += bars[stock][0].c
-    resp.append(totalPrice)
-
-  # Submit a batch order that returns completed and uncompleted orders.
-  def sendBatchOrder(self, qty, stocks, side, resp):
-    executed = []
-    incomplete = []
-    for stock in stocks:
-      if(self.blacklist.isdisjoint({stock})):
-        respSO = []
-        tSubmitOrder = threading.Thread(target=self.submitOrder, args=[qty, stock, side, respSO])
-        tSubmitOrder.start()
-        tSubmitOrder.join()
-        if(not respSO[0]):
-          # Stock order did not go through, add it to incomplete.
-          incomplete.append(stock)
+                continue
+
+        # Determine amount to long/short based on total stock price of each bucket.
+        equity = int(float(self.alpaca.get_account().equity))
+
+        self.short_amount = equity * 0.30
+        self.long_amount = equity - self.short_amount
+
+        resp_get_tp_long = self.get_total_price(self.long)
+
+        resp_get_tp_short = self.get_total_price(self.short)
+
+        self.q_long = int(self.long_amount // resp_get_tp_long)
+        self.q_short = int(self.short_amount // resp_get_tp_short)
+
+    # Get the total price of the array of input stocks.
+    def get_total_price(self, stocks) -> float:
+        total_price = 0
+        for stock in stocks:
+            bars = self.alpaca.get_bars(stock, TimeFrame.Minute,
+                                        datetime.date.today().isoformat(),
+                                        datetime.date.today().isoformat(), limit=1,
+                                        adjustment='raw')
+
+            total_price += bars[stock][0].c
+        return total_price
```

Comment:
While this logic is correct, I think using the latest trade price is much more accurate. The price can move a lot since the last minute bars closing price.
```suggestion
        return sum(self.alpaca.get_latest_trade(stock).price for stock in stocks)
```